demo/admin/src/pages/pageTreeCategories.tsx

```diff
@@ -0,0 +1,14 @@
+import { AllCategories } from "@comet/cms-admin";
```

Comment:
Is this the best location for this file? Where would you put it in Starter (folder structure is different -> we should align it in Demo)

Reply:
`src/pages` previously existed in Starter. We could readd it. 

None of the folders in the Starter are really fitting. We could either create a dedicated folder (e.g. `pageTreeCategories`) or put it directly in `admin/src` folder

Reply:
pageTree is not pages, pages is jsut one (of many) page types that are placed in the pageTree

Reply:
We could also put it in `src/documents`

Reply:
`src/pageTree`